lib/language_pack/helpers/bundler_wrapper.rb

```diff
@@ -37,7 +38,16 @@ def install
 
   def clean
     ENV['BUNDLE_GEMFILE'] = @orig_bundle_gemfile
-    FileUtils.remove_entry_secure(bundler_path) if Dir.exist?(bundler_path)
+    FileUtils.remove_entry_secure(@bundler_tmp) if Dir.exist?(@bundler_tmp)
+
+    if LanguagePack::Ruby::BUNDLER_VERSION  == "1.7.12"
+      # Hack to cleanup after pre 1.8 versions of bundler. See https://github.com/bundler/bundler/pull/3277/
+      Dir["#{Dir.tmpdir}/bundler*"].each do |dir|
+        FileUtils.remove_entry_secure(dir) if Dir.exist?(dir) && File.stat(dir).writable?
+      end
+    else
+      raise "Bundler cleanup hack is out of date, please update it to be compatible with #{LanguagePack::Ruby::BUNDLER_VERSION}"
```

Comment:
We're about to test run 1.9.6. We will have to take this raise out. How critical is this behavior for you? Should we remove the raise or can I remove it all? Was going to merge 1.9.6 into master sometime this week, shoot to deploy thursday if no issues.